storagenode/console/bandwidth.go

```diff
@@ -40,8 +37,8 @@ type Ingress struct {
 type BandwidthInfo struct {
 	Egress    Egress  `json:"egress"`
 	Ingress   Ingress `json:"ingress"`
-	Used      int64   `json:"used"`
-	Remaining int64   `json:"remaining"`
+	Used      float64 `json:"used"`
+	Available float64 `json:"remaining"`
```

Comment:
Either change this back to Remaining or adjust the json name 👍

Reply:
good catch